src/screens/IndexScreen.js

```diff
@@ -210,9 +210,8 @@ export const IndexScreen = ({ navigation, route }) => {
           route={route}
           position={position}
           queryVariables={{
-            category: queryVariables.category,
-            categoryId: queryVariables.categoryId,
-            dataProvider: queryVariables.dataProvider
+            ...queryVariables,
+            limit: undefined
```

Comment:
is `limit` already included in `queryVariables` and needs to be overwritten?

Reply:
correct. that was the initial reason for selecting specific query variables there.
back then there were less query variables, so selecting seemed reasonable.

it turned out that even though new ones got added, the only one we want to keep omitting is the limit. so i changed it to overwrite the limit instead